ftw/mail/utils.py

```diff
@@ -11,6 +11,7 @@
 IMG_SRC_RE = re.compile(r'<img[^>]*?src="cid:([^"]*)', re.IGNORECASE|re.DOTALL)
 BODY_RE = re.compile(r'<body>(.*)</body>', re.IGNORECASE|re.DOTALL)
 APPLE_PARTIAL_ENCODING_RE = re.compile(r'^"(.*=\?.*\?=.*)"( <.*>)$')
+ENCODED_WORD_WITHOUT_LWSP = re.compile(r'(=\?.*?\?=)(\r\n)')
```

Comment:
@jone `linear-white-space` is `[CRLF] LWSP` - so the `CRLF` is optional. For now I made the regex match the exact pattern that I encountered in the wild and was causing problems - but you're right, for the general case your suggested pattern would be correct. However the LWSP part after the CRLF would need to be grouped so that I can substitute either SPACE or HTAB back in (whichever one it was) while still replacing CRLF with SPACE.

So in theory `(\r\n)([ \t])` should work, and tests should still pass without a change :wink: